print-medsa.c

```diff
@@ -190,3 +202,59 @@ medsa_print(netdissect_options *ndo,
 trunc:
 	nd_print_trunc(ndo);
 }
+
+/* The DSA tag is 4 bytes. It has the same content as the last 4 bytes
```

Comment:
*Last* 4 bytes, or *first* 4 bytes?  It looks, from the definition of `struct mdsa_pkthdr`, as if the EDSA tag has 4 bytes of tag and 2 bytes of Ethernet type.

Reply:
Last.

EDSA has 2 bytes of Ethertype, 2 bytes reserved, and then 4 bytes which are identical to a DSA tag.
For printing EDSA, when medsa_print() is called, it is because the Ethertype has been matched. The Ethertype has already been pulled off, so medsa_print() only sees 6 bytes of the 8 byte header.
After the EDSA or DSA header comes the real ethertype value for the frame.  So for EDSA, this is the second Ethertype header in the frame.

Reply:
Since the ether_type seems to be causing confusion, i removed it from the structure.

The broadcom cut/paste problem has also been fixed.